### PR TITLE
Fix error in userHelperDebug

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #25401 .

### Summary of Changes
$action is a object, not an array

### Testing Instructions
See description in #25401

### CODE REVIEW

File: administrator/components/com_users/Helper/UsersHelperDebug.php [hunk 1]
```diff
@@ -88,11 +88,11 @@ public static function getDebugActions($component = null)
...
 				foreach ($component_actions as &$action)
 				{
-					$descr = (string) $action['title'];
+					$descr = (string) $action->title;
 
-					if (isset($action['description']) && !empty($action['description']))
+					if (!empty($action->description))
 					{
-						$descr = (string) $action['description'];
+						$descr = (string) $action->desctription;
```
Comment (SharkyKZ) [inline administrator/components/com_users/Helper/UsersHelperDebug.php]:
```suggestion
						$descr = (string) $action->description;
```

**Comment by SharkyKZ** [comment]
Some more changes are needed below, see original PR https://github.com/joomla/joomla-cms/pull/22834/files#diff-85cbdec3db954ea40d4b3f3713b2caaaR122.

**Comment by chmst** [comment]
thank you, @SharkyKZ

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 261c9dccebd113239869a612045b958b3c55a509<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25402">issues.joomla.org/tracker/joomla-cms/25402</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 261c9dccebd113239869a612045b958b3c55a509<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25402">issues.joomla.org/tracker/joomla-cms/25402</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by HLeithner** [comment]
thx

**Pull request merged**